uml/backend-software-system/screen-handler.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
+
+package "Screen Handler" {
+  class EventType {
+    type : String
+    sprite : Sprite
+  }
+
+  class SpritesReader {
+    +readSprites() : List<Sprite>
+  }
+
+  class ScreenManager {
+    -sprites : List<Sprite>
```

Comment:
So it's the GUI component expecting Sprite objects to render?

Reply:
Given an input listener, ScreenManager will change the Sprite that is been shown to a new one. In short a signal will let ScreenMapper to update for expected Sprite. But reviewing the "Input Device Monitor" package, ISender should be part of ScreenManager here due to it is receiving a WindowEvent as parameter to show the new Sprite I guess.

Reply:
There were some new changes on the design of the UML for GUI and these old classes doesn't form part now, but it helped to design the new UML.